website/source/docs/upgrade/upgrade-specific.html.md

```diff
@@ -51,6 +51,16 @@ Raft Protocol versions supported by each Consul version:
 In order to enable all [Autopilot](/guides/cluster/autopilot.html) features, all servers
 in a Nomad cluster must be running with Raft protocol version 3 or later.
 
+### Node Draining Improvements
+
+Node draining via the [`node drain`][drain-cli] command or the [drain
+API][drain-api] has been substantially changed in Nomad 0.8. In Nomad 0.7.1 and
+earlier draining a node would immediately stop all allocations on the node
+being drained. Nomad 0.8 now supports a [`migrate`][migrate] stanza in job
+specifications to control how many allocations may be migrated at once.
```

Comment:
Maybe specify the advantage of this unless the decommissioning nodes guides does so (i.e, to ensure a service always is alive even as nodes are being decommissioned).